build.gradle

```diff
@@ -105,7 +105,7 @@ subprojects { project ->
             it.useTarget group: 'cglib', name: 'cglib', version: '3.2.0'
           }
           if (it.requested.group == 'com.google.guava') {
-            it.useTarget group: 'com.google.guava', name: 'guava', version: '18.0'
+            it.useTarget group: 'com.google.guava', name: 'guava', version: '20.0'
```

Comment:
If the specific reason we pinned to `18.0` is gone, it might make sense to just entirely remove the version pinning here so things inherit our usual period version bumps.